### ISSUE TITLE
Bug: Make Time Period Error Message  Compatible with Screen Readers

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#1549]
## Required Fix 
- [x] RF-11

## Context
See [EPA CAMD Accessibility Report ](https://teams.microsoft.com/l/file/31E8B6EF-1313-4495-B867-46347BB7E4A2?tenantId=88b378b3-6748-4867-acf9-76aacbeca6a7&fileType=docx&objectUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam%2FShared%20Documents%2FDatateers%2F5-UI%20Design%2F02%20508%20and%20Accessibility%2FEPA.CAMPD.Accessibility.Report%20.docx&baseUrl=https%3A%2F%2Fusepa.sharepoint.com%2Fsites%2FCAMDCVPTeam&serviceName=teams&threadId=19:f4349e2ba95e43f3b29e8208bc8f5d5f@thread.skype&groupId=21a5e1cd-d3f1-48a6-801f-0d34e2d63d23) for how to fix and additional details 

| Issue #  | Issue Description  |
|-|-|
| RF-11 | The messages shown to the user as a result of input errors in form fields on the page are not accessible to Assistive Technology. This issue is a violation of Section 508 and of Success 3.3.1 in WCAG 2.0. |

NVDA and Jaws can not read the checkmark
![image.png](https://images.zenhubusercontent.com/5f5928c1045cffdce7ce89c3/5c204b95-5f8f-41c1-aad7-b34bce970300)

Solution: Only display the error messages that do not have a check mark

**Comment by aprematta** [comment]
This has been tested with all time period messages and passes. Check marked items no longer appear and focus shifts correctly to the error message.
